ci/unit/test_nlt_node.sh

```diff
@@ -7,6 +7,8 @@ set -uex
 
 sudo bash -c 'echo 1 > /proc/sys/kernel/sysrq'
 sudo mkdir -p /mnt/daos
+# using mmap()'ed ULT stacks requires to bump system default
+sudo sysctl vm.max_map_count=1000000
```

Comment:
hm, is that something that we need to handle in systemd (if possible)?

Reply:
ok, done.